javascript/dwa-starter-vue/tests/App.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from 'vitest'
+
+import { mount } from '@vue/test-utils'
+import HelloWorld from '../src/App.vue'
+
+describe('HelloWorld', () => {
+  it('renders properly', () => {
+    const wrapper = mount(HelloWorld)
+    expect(wrapper.text()).toContain('Hello World!')
```

Comment:
@Johnnyevans32  oopsie. either fix here or in the page! 😛 the page is saying `Hello world!` (lowercase `w`)

Reply:
oh must be some typo somewhere, fixed now

Reply:
@leordev